frontend/src/metabase/dashboard/actions/getNewCardUrl.unit.spec.ts

```diff
@@ -0,0 +1,70 @@
+import {
```

Comment:
Tests moved from `frontend/src/metabase/parameters/utils/dashboards.unit.spec.js` and updated to use TS.